Clinical-Genomics/cg
# Changed qc target from application target reads to expected reads

## Description

For NIPT uploads we have a certain amount of reads needed for delivery.. We are following the application versions expected reads for delivery and should thereby use that as a threshold instead of the target reads of the application.

### Changed

- QC check on target reads to expected reads

### How to prepare for test
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix/expected-reads-nipt-upload`

### How to test

### Expected test outcome

## Review
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

## Implementation Plan
